site/tasks/ipam/stop-remove-peers-ipam.md

```diff
@@ -37,22 +37,20 @@ name. Alternatively, you can supply a peer name as shown in `weave status`.
 
 ### <a name="caution-rmpeer"></a>Caution###
 
-You cannot call `weave rmpeer` on more than one host. The address
-space, which was owned by the stale peer cannot be left dangling, and
-as a result it gets reassigned. In this instance, the address is
-reassigned to the peer on which `weave rmpeer` was run. Therefore, if
-you run `weave forget` and then `weave rmpeer` on more than one host
-at a time, it results in duplicate IPs on more than one host.
-
-Once the peers detect the inconsistency, they log the error and drop
+You must not invoke `weave rmpeer` for the same peer on more than one
+host. The removed peer's address range cannot be left dangling and is
```

Comment:
You could make the first sentence stronger by saying:

Do not invoke `weave rmpeer` for the same peer on more than one host.